.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+name: Run All Tests
+
+on:
+  pull_request:
+    types: [
+      opened,
+      labeled,
+      ready_for_review,
+      reopened,
+      synchronize,
+    ]
+    branches:
+      - main
+      - release**
+      - production**
+      - hotfix**
+      - lab**
+
+env:
+  NODE_VERSION: '14.x'
+
+jobs:
+  Type:
+    if: |
+      github.event.pull_request.draft == false ||
+      (
+        github.event.action == 'labeled' &&
+        contains(github.event.pull_request.labels.*.name, 'run-all-test')
+      )
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+
+    - uses: actions/cache@v2
+      id: yarn-cache
+      with:
+        path: |
+          ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          **node_modules
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
```

Comment:
I just noticed I shouldn't set `restore-keys` with hash
Reference: https://github.community/t/understanding-cache-action-keys/17503/2

Reply:
so `restore-key` is like a fall-back / backup plans for the cache key?